src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodBase.cs

```diff
@@ -149,7 +149,7 @@ protected static async Task NotSupported_ExtractMethodAsync(string codeWithMarke
                 result.SucceededWithSuggestion ||
                 (allowBestEffort && result.Status.HasBestEffort()));
 
-            var doc = result.Document;
+            var (doc, _) = await result.GetFormattedDocumentAsync(CancellationToken.None);
```

Comment:
not a comment on this PR, but i cannot understand why this test harness is so complex.

Reply:
Yes, it seems crazy but there are multiple entry points to this feature so maybe it's related to that?